docs/usage/annotations.rst

```diff
@@ -0,0 +1,118 @@
+Annotations
+===========
+
+A developer can create mappings between a concrete implementation (openqasm) and a program modelled in the `frontend <https://github.com/LEQO-Framework/low-code-modeler>`_ by using various openqasm3 annotations.
+
+Annotations can be emulated in openqasm2 by using special comments.
+
+.. warning::
+    The `whole line <https://openqasm.com/language/directives.html#annotations#:~:text=continue%20to%20the%20end%20of%20the%20line>`_ will be interpreted like an annotation.
+    Therefore you cannot use inline-comments on annotations!
+
+Input
+-----
+
+One input is defined as a single qubit register (:class:`~openqasm3.ast.QubitDeclaration`) with a single `@leqo.input` annotation.
```

Comment:
```suggestion
One input is defined as a single qubit register (:class:`~openqasm3.ast.QubitDeclaration`) with a single `@leqo.input` annotation followed by the index of that input.
The input index ...
```
Format (i.e., a single positive? integer number) and **purpose** of the input index should be presented here already (at least briefly). Not everyone will read the bullet points below carefully.